ui/src/Drawer.tsx

```diff
@@ -55,28 +56,32 @@ export class Drawer extends React.PureComponent {
           }
 
           return (
-            <MuiDrawer
-              className="drawer"
-              variant="persistent"
-              anchor="right"
-              open={drawerState !== "closed"}
-            >
-              <div className="drawer__header">
-                <div>
-                  <IconButton onClick={() => setDrawerState("closed")}>
-                    <ChevronRightIcon />
-                  </IconButton>
+            <ClickAwayListener onClickAway={() => {
+              setDrawerState('closed');
+            }}>
+              <MuiDrawer
+                className="drawer"
+                variant="persistent"
+                anchor="right"
+                open={drawerState !== "closed"}
+              >
+                <div className="drawer__header">
+                  <div className="drawer__close_icon MuiButton-contained">
+                    <IconButton onClick={() => setDrawerState("closed")}>
```

Comment:
Small style comment: I notice "closed" and 'closed' on Lines 70 and 60 have different quotation marks. Do you have Typescript formatting turned on in your VSCode? If not, can you turn it on so we have a consistent auto-formatter? One helpful feature is VSCode's save-on-format feature which will take care of this on save.

Reply:
I have it turned on but for some reason the format on save feature is not working.... will check it out :/